homeassistant/helpers/selector.py

```diff
@@ -154,7 +151,7 @@ class BaseSelectorConfig(TypedDict, total=False):
         vol.Optional("device_class"): vol.All(cv.ensure_list, [str]),
         # Features supported by the entity
         vol.Optional("supported_features"): [
-            vol.All(cv.ensure_list, [str], _validate_supported_features)
+            vol.Any(int, vol.All(cv.ensure_list, [str], _validate_supported_features))
```

Comment:
Why would we accept integers and not rely on `_validate_supported_features` resolving it?